addons/project/static/src/views/project_task_list/project_task_list_controller.js

```diff
@@ -1,43 +0,0 @@
-/** @odoo-module */
-
-import { useService } from '@web/core/utils/hooks';
-import { ListController } from '@web/views/list/list_controller';
-
-export class ProjectTaskListController extends ListController {
-    setup() {
-        super.setup();
-        this.taskRecurrence = useService('project_task_recurrence');
-    }
-
-    getStaticActionMenuItems() {
-        const menuItems = super.getStaticActionMenuItems();
-        const hasAnyRecurrences = this._anySelectedTasksWithRecurrence();
-
-        if (hasAnyRecurrences) {
```

Comment:
So, we no longer need to have a custom code when we remove/archive a recurrence task in the list view? 🤔 
If yes then cool 🤩

Reply:
No, we can delete every task of the recurrence, because we still have the template :)